http/src/database.ts

```diff
@@ -0,0 +1,16 @@
+import { User } from "./interfaces";
```

Comment:
please, create a separate repository, e.g. DataRepository, move it to the repositories folder, and create appropriate methods for CRUD

Reply:
done